guides/common/Makefile

```diff
@@ -24,6 +25,10 @@ ifeq ($(UNAME), Darwin)
 BROWSER_OPEN = open
 endif
 
+ifeq ($(LINKS), LOCAL)
+BASEURL_ATTR = "-a BaseURL=file://$(BUILD_DIRA)"
+endif
+
```

Comment:
When `LINKS=LOCAL` is passed, it adds the local BaseURL. Otherwise it builds as usual.